services/storage-users/pkg/command/trash_bin.go

```diff
@@ -53,3 +89,402 @@ func PurgeExpiredResources(cfg *config.Config) *cli.Command {
 		},
 	}
 }
+
+func listTrashBinItems(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:      "list",
+		Usage:     "Print a list of all trash-bin items of a space.",
+		ArgsUsage: "['spaceID' required]",
+		Flags:     []cli.Flag{},
```

Comment:
Maybe we can add a `--json` flag that prints a `json` instead of a human readable string? That way I can pipe the output in another command.